### ISSUE TITLE
Plugin that completely hides DMs from muted users

### ISSUE THREAD

**Issue by Tenrys** [Vencord/plugin-requests#812]
### Plugin Description

I want to be able to make it so when I mute a friend/user, their DMs won't pop up at the top again when they send something else if I close the DMs altogether. I don't want to have to block them completely, just take a break from their existence for a while.

### Request Agreement

- [X] I read the above and made sure my plugin is possible and allowed
- [X] I checked the [existing plugins](https://vencord.dev/plugins) and made sure my plugin doesn't already exist
- [X] I checked every box, despite not reading any
- [X] I [searched the existing issues](https://github.com/Vencord/plugin-requests/issues?q=is%3Aissue) and made sure a similar request doesn't already exist

**Comment by Scyye** [comment]
![image](https://github.com/Vencord/plugin-requests/assets/97131358/cedc0983-8a30-47be-9480-850b98c196cc)
nice

**Comment by Tenrys** [comment]
better?

**Comment by Scyye** [comment]
well, did you read and follow them?

> better?

**Comment by Tenrys** [comment]
yes I searched for existing plugins, and looked briefly for similar issues already. I can read, I frequently check for new plugins as well, my bad that I lost attention for 2 seconds I guess despite reading the rest I guess.

**Comment by cheesesamwich** [comment]
[This can achieve something similar](https://github.com/cheesesamwich/Tobleronecord/blob/main/src/tobleroneplugins/ClientSideBlock.tsx)

**Comment by Tenrys** [comment]
how do I install that to vencord? I don't want to install something separate

**Comment by Scyye** [comment]
3rd party plugins arent supported.
